### PR TITLE
[Oxfordshire] Alloy integration

### PR DESCRIPTION
**Pull request by davea**
For sending street lighting reports to Alloy, and displaying existing defects on the map.

[skip changelog]

### CODE REVIEW

File: templates/web/hackney/footer_extra_js.html [hunk 1]
```diff
@@ -1,7 +1 @@
-[%
-IF bodyclass.match('mappage');
-  scripts.push(
-    version('/cobrands/fixmystreet-uk-councils/alloy.js'),
-  );
-END %]
-[% PROCESS 'footer_extra_js_base.html' highways=1 tfl=1 cobrand_js=1 validation=1 %]
+[% PROCESS 'footer_extra_js_base.html' highways=1 tfl=1 cobrand_js=1 validation=1 alloy=1 %]
```
Comment (struan) [inline templates/web/hackney/footer_extra_js.html:1]: nice

**Review by struan — APPROVED** [review]
👍 

Nice clear self contained commits \o/

File: templates/web/hackney/footer_extra_js.html (hunk 1 above)
Comment (davea) [reply templates/web/hackney/footer_extra_js.html:1]: That's @dracos's work, I should add `Co-authored-by:` really!

**Comment by dracos** [comment]
@davea Could you look over my commits, I've looked over yours :) Nothing major.

**Pull request merged**